pkg/koyeb/instances_exec.go

```diff
@@ -1,27 +1,18 @@
 package koyeb
 
 import (
-	"context"
-	"encoding/base64"
-	"fmt"
-	"io"
-	"net/http"
-	"net/url"
 	"os"
-	"os/signal"
-	"syscall"
-	"time"
 
-	"github.com/gorilla/websocket"
-	"github.com/koyeb/koyeb-api-client-go/api/v1/koyeb"
-	"github.com/moby/term"
-	"github.com/pkg/errors"
-	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+
+	"github.com/koyeb/koyeb-api-client-go/api/v1/koyeb"
 )
 
 func (h *InstanceHandler) Exec(cmd *cobra.Command, args []string) error {
-	returnCode, err := h.exec(cmd, args)
+	returnCode, err := exec(ExecId{
+		Id:   h.ResolveInstanceArgs(args[0]),
+		Type: koyeb.EXECCOMMANDREQUESTIDTYPE_INSTANCE_ID,
+	}, args)
```

Comment:
I think that we need to give `args[1:]` to the `exec` function here, because `args[0]` is the ID of the resource we want to exec on 🙂